server/channels/app/channel_bookmark_test.go

```diff
@@ -125,9 +129,14 @@ func TestUpdateBookmark(t *testing.T) {
 			HasPreviewImage: true,
 		}
 
-		th.App.Srv().Store().FileInfo().Save(th.Context, file2)
-		th.App.Srv().Store().FileInfo().AttachToPost(th.Context, file2.Id, model.NewId(), th.BasicChannel.Id, model.BookmarkFileOwner)
-		defer th.App.Srv().Store().FileInfo().PermanentDelete(th.Context, file2.Id)
+		_, err = th.App.Srv().Store().FileInfo().Save(th.Context, file2)
+		assert.NoError(t, err)
+		err = th.App.Srv().Store().FileInfo().AttachToPost(th.Context, file2.Id, model.NewId(), th.BasicChannel.Id, model.BookmarkFileOwner)
+		assert.NoError(t, err)
+		defer func() {
+			err = th.App.Srv().Store().FileInfo().PermanentDelete(th.Context, file2.Id)
+			assert.NoError(t, err)
+		}()
 
 		bookmark2.FileId = file2.Id
 		bookmarkResp, err = th.App.CreateChannelBookmark(th.Context, bookmark2, "")
```

Comment:
This also needs to be renamed to `*model.AppError`.

Reply:
For this too? https://github.com/mattermost/mattermost/pull/28789#discussion_r1816418394

Reply:
Yes, this comment in related to https://github.com/mattermost/mattermost/pull/28789#discussion_r1816418394

Reply:
Fixed!

Reply:
Sorry, this is still unresolved.